_data/simple-icons.json

```
@@ -1871,6 +1871,11 @@
             "hex": "0052CC",
             "source": "https://www.atlassian.design/guidelines/marketing/resources/logo-files"
         },
+        {
+            "title": "Bambu Lab",
+            "hex": "00ae42",
+            "source": "https://github.com/bambulab/BambuStudio/blob/master/resources/images/BambuStudio.svg" 
```

There is also an SVG available on the homepage of the website. I'd suggest we use that as a source.
```suggestion
            "source": "https://bambulab.com" 
```